### DISCUSSION TITLE
ar state with scene viewer

### DISCUSSION THREAD

**Discussion by devhims** [google/model-viewer#2066] [Q&A]
As per the documentation, both `ar-status` attribute and event values are valid for WebXR mode only. Whereas on SceneViewer, we only get `ar-status` value of `not-presenting`. It's for sure implemented in SceneViewer as we get a prompt to move phone when the session starts. So is it possible to access these values in Scene Viewer and Quick Look modes?

**Comment by devhims** [comment]
The feature I'm trying to implement aims to provide feedback to users after the button press before SceneViewer or QuickLook gets launched. The issue is I'm unable to reset the loading state of the button after the user cancels the session.  

![tempFileForShare_20210301-235134](https://user-images.githubusercontent.com/17825870/109541372-bf46f800-7ae9-11eb-9f23-75bd793b9b30.jpg)

**Reply by devhims** [reply]
Hi @elalish, do you have any inputs to resolve this?

**Reply by elalish** [reply]
Yes, this is why we are pushing the in-browser WebXR mode. Because of security isolation we get almost no feedback about what happens with an app intent, or even when we've returned. Native apps just don't play well with web browsers. If you come up with a solution, feel free to contribute it, but we have yet to find anything reliable.

**Reply by devhims** [reply]
Got it! I installed model viewer v 1.5.0 from npm, but it didn't support AR in WebXR mode during testing. I'm not 100% sure, so will retry tomorrow. Any chance this is an intentional change?

**Reply by elalish** [reply]
Are you using an up-to-date Chrome on Android? If WebXR is not working but SceneViewer is, then please file a bug with your device/OS details.

**Reply by devhims** [reply]
@elalish The issue is that https is required for WebXR, whereas SceneViewer and QuickLook work fine on http. Once I configured my dev server to use https, WebXR mode worked just fine.
